### PR TITLE
Menu update

### PR DESCRIPTION
**Pull request by clintonb**
Updated Site-wide Navigation Bar
- Previously jQuery and Bootstrap were only loaded when pages explicitly loaded them via RequireJS. These libraries are now loaded for every page. This resolves an issue seen on the course index page (which has no JS needs) where the main nav dropdown was broken since Bootstrap was not loaded.
- Updated existing pages to simply load their respective scripts. The base page will handle loading RequireJS.
- Help item is now at top-level nav, instead of nested in the user dropdown.

Review/merge #48 first.

@dsjen @rocha

### CODE REVIEW

**Comment by clintonb** [comment]
screen shot 2014-08-22 at 2 37 13 pm

**Comment by rocha** [comment]
LGTM :rocket:

File: analytics_dashboard/static/js/application-main.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Load scripts needed across the application.
+ */
+
+require(['jquery', 'bootstrap'], function($, bootstrap){
```
Comment (dsjen) [inline analytics_dashboard/static/js/application-main.js]: I don't believe jquery is needed here (bootstrap already has a dependency on jquery).

Comment (clintonb) [inline analytics_dashboard/static/js/application-main.js]: Updated.

**Comment by dsjen** [comment]
Looks great!

**Pull request merged**